templates/migration/templates/migrationservice-configmap.yaml

```diff
@@ -18,6 +18,14 @@ data:
   USERS_PUBLIC_PLAN_TECHNICAL_PASSWORD: "{{ .Values.publicPlanTechnicalPassword }}"
   USERS_PUBLIC_PLAN_TECHNICAL_EMAIL: "{{ .Values.publicPlanTechnicalEmail }}"
   USERS_PUBLIC_PLAN_TECHNICAL_TENANTID: "{{ .Values.publicPlanTechnicalTenantId }}"
+
+  WEBLATE_CONFIG_SERVER_URL: "{{ .Values.weblateServerUrl }}"
```

Comment:
And antoher one: ;-) 
`migration.weblate.api.url` instead of `migration.weblate.server.url` would be good for the url because in mailservice you used "api" instead of "server" so we have the same naming for the same stuff ;-)